internal/k8s/controller.go

```diff
@@ -277,20 +277,27 @@ func NewLoadBalancerController(input NewLoadBalancerControllerInput) *LoadBalanc
 		lbc.externalDNSController = ed_controller.NewController(ed_controller.BuildOpts(context.TODO(), lbc.namespaceList, lbc.recorder, lbc.confClient, input.ResyncPeriod, isDynamicNs))
 	}
 
+	glog.V(3).Infof("Nginx Ingress Controller has class: %v", input.IngressClass)
+
+	collectorConfig := telemetry.CollectorConfig{
```

Comment:
collector config needs to be created only when user opts in, correct? if yes, maybe to move it after L289 - `if input.EnableTelemetryReporting` and take care about config only when user wants or not telemetry?